### PR TITLE
Add dask-ctl package

### PR DESCRIPTION
**Pull request by jacobtomlinson**
Add [dask-ctl](https://pypi.org/project/dask-ctl/).

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
@conda-forge/help-python this is ready for review.

cc @jakirkham

File: recipes/dask-ctl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "dask-ctl" %}
+{% set version = "2021.1.0a0" %}
```
Comment (xhochy) [inline recipes/dask-ctl/meta.yaml:2]: This looks like a pre-release version, is this correct? We only upload relases on conda-forge.

Comment (jakirkham) [reply recipes/dask-ctl/meta.yaml:2]: Could we just add a `skip` for now and then follow [CFEP-05]( https://github.com/conda-forge/cfep/blob/master/cfep-05.md ) on the feedstock? Also somewhat curious whether the same exception used for `black` might apply here.

Comment (isuruf) [reply recipes/dask-ctl/meta.yaml:2]: CFEP-05 can be followed here. No need to `skip`.

Comment (jacobtomlinson) [reply recipes/dask-ctl/meta.yaml:2]: I'm also happy to hold off for a release, pre-releases will be rare with this package. I expect the first full release will be in the next week or so. I was just trying to get out ahead of things.

Comment (jacobtomlinson) [reply recipes/dask-ctl/meta.yaml]: We have now had a full release so I've updated this PR to point to that.

**Review by jakirkham — APPROVED** [review]
LGTM. Thanks for working on this Jacob! 😄

**Comment by jacobtomlinson** [comment]
ping @xhochy @isuruf @jakirkham 

Is it possible to get this merged?

**Comment by jacobtomlinson** [comment]
Thanks @xhochy

**Comment by jakirkham** [comment]
Thanks Jacob! 😀

**Pull request merged**